go_expvar/datadog_checks/go_expvar/data/conf.yaml.example

```diff
@@ -69,3 +69,20 @@ instances:
     #     alias: go_expvar.my_custom_name
     #     type: gauge
     #   - path: routes/get_.*/count
+    
+    ## @param skip_proxy - boolean - optional - default: false
```

Comment:
but is this integration using self.http?

Reply:
Seems so: https://github.com/DataDog/integrations-core/pull/3743/files#diff-145bbbf6343051c2b7abb8224837741dR81